src/frontend/components/DashboardVideoPane/index.tsx

```diff
@@ -143,7 +155,7 @@ export const DashboardVideoPane = ({ video }: DashboardVideoPaneProps) => {
   };
 
   useEffect(() => {
-    if ([PENDING, UPLOADING, PROCESSING].includes(video.upload_state)) {
+    if ([PENDING, PROCESSING].includes(video.upload_state)) {
```

Comment:
I think the `PENDING` test is useless. We now the exact state with the `uploadManagerState`. We can start polling once the upload in sucess and also when the video state is in `PROCESSING`. WDYT ?

Reply:
yes :+1: